### PR TITLE
01 get/api/categories

### PR DESCRIPTION
**Pull request by tuttitaken01**
Task GET categories done

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -32,8 +32,13 @@ describe("1.getCategories", () => {
             .then((res) => {
                 let categories = res.body.categories;
                 expect(Array.isArray(categories)).toBe(true);
+                categories.forEach(cat => {
```
Comment (Hard-Luck) [inline __tests__/app.test.js:44]: We should test the length of the array, as this is done with test data we should know what were getting back.
This also protects the forEach as that would pass if an empty array is passed.

File: app.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const {
 } = require('./controllers.js')
 
 app.get('/api', serverStatus);
-//app.get('/api/categories', getCategories);
+app.get('/api/categories', getCategories);
 
 /*const { PORT = 9090 } = process.env;
```
Comment (Hard-Luck) [inline app.js:19]: shouldn't have commented code in here. 
Look to put the listen into a separate file.

Comment (Hard-Luck) [reply app.js:19]: Should have a path that catches all non match paths and sends a 404

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
Some changes required

**Review by MrMcSnail — COMMENTED** [review]
Good stuff

**Pull request merged**